configure.ac

```diff
@@ -190,11 +190,15 @@ AC_ARG_ENABLE([no-stack],
 
 AC_ARG_ENABLE([upstart],
     [  --enable-upstart
-       Do not build support for the Upstart init system [default=yes]])
+       Enable support for the Upstart init system [default=try]],
+    [],
+    [enable_upstart=try],
+)
 
+dnl FIXME currently unused
```

Comment:
I guess this was a problem with b763e8fa? Maybe just add

    -a "x${enable_systemd}" != xno

before setting HAVE_systemd = 1?